src/server.js

```diff
@@ -1,12 +1,25 @@
 import express from 'express'
-import api from './routes/api'
-import models, { connectDb } from './schema/models'
+// import models, { connectDb } from './schema/models'
```

Comment:
same message as above

Reply:
I leave that there to use it as an example on how to use router, but ok